host/src/attribute.rs

```diff
@@ -24,45 +24,85 @@ pub const GENERIC_ATTRIBUTE_UUID16: Uuid = Uuid::Uuid16(0x1801u16.to_le_bytes())
 
 #[derive(Debug, Clone, Copy)]
 #[repr(u8)]
+/// An enum of possible characteristic properties
+/// 
+/// Ref: BLUETOOTH CORE SPECIFICATION Version 6.0, Vol 3, Part G, Section 3.3.1.1 Characteristic Properties
```

Comment:
I would like to propose this syntax for referencing Bluetooth core specification. This should help other devs cross reference things.